python-wheel.cmake

```diff
@@ -57,6 +58,10 @@ function (add_wheel WHEEL_TARGET)
     set(WHEEL_URL "")
   endif()
 
+  if (NOT WHEEL_URL)
```

Comment:
I believe this check is supposed to be for WHEEL_EMAIL?

Reply:
Yes, Thanks. I forgot to change that.